### PR TITLE
Documentation for Seq.contains + small typo fix to Seq.tail docs

### PR DESCRIPTION
**Pull request by liboz**
As title. Related to #184

### CODE REVIEW

File: docs/conceptual/seq.contains['t]-function-[fsharp].md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+# Seq.contains<'T> Function (F#)
+
+Evaluates to **true** if the given element is in the given sequence.
+
+**Namespace/Module Path:** Microsoft.FSharp.Collections.Seq
+
+**Assembly:** FSharp.Core (in FSharp.Core.dll)
+
+
```
Comment (cartermp) [inline docs/conceptual/seq.contains['t]-function-[fsharp].md]: Nit: extraneous space (even though it renders just fine)

File: docs/conceptual/seq.contains['t]-function-[fsharp].md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+#### Parameters
+*element*
+Type: **'T**
+
+The element to find.
+
+*source*
+Type: [seq](https://msdn.microsoft.com/library/2f0c87c6-8a0d-4d33-92a6-10d1d037ce75)**&lt;'T&gt;**
+
+
```
Comment (cartermp) [inline docs/conceptual/seq.contains['t]-function-[fsharp].md]: Nit: extraneous space

File: docs/conceptual/seq.contains['t]-function-[fsharp].md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+```
+true
+false
+```
+
+## Platforms
+Windows 8, Windows 7, Windows Server 2012, Windows Server 2008 R2
+
+
```
Comment (cartermp) [inline docs/conceptual/seq.contains['t]-function-[fsharp].md]: Nit: extraneous space

File: docs/conceptual/seq.contains['t]-function-[fsharp].md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+## Version Information
+**F# Core Library Versions**
+
+Supported in: 4.0, Portable
+
+
```
Comment (cartermp) [inline docs/conceptual/seq.contains['t]-function-[fsharp].md]: Nit: extraneous spaces

File: docs/conceptual/snippets/fssequences/snippet205.fs [hunk 5]
```diff
@@ -0,0 +1,8 @@
+let seq1 = seq {1..10}
```
Comment (cartermp) [inline docs/conceptual/snippets/fssequences/snippet205.fs]: Nit: spaces after/before the brackets is generally preferred: `seq { 1..10 }`

**Comment by cartermp** [comment]
Left nitpicky feedback, but otherwise this looks good.

File: docs/conceptual/seq.contains['t]-function-[fsharp].md (hunk 1 above)
Comment (dend) [inline docs/conceptual/seq.contains['t]-function-[fsharp].md]: Can we also wrap **true** into `true` (use single backticks)?

**Comment by cartermp** [comment]
LGTM.  Thanks, @liboz!

**Comment by dend** [comment]
LGTM 🔥

**Pull request merged**